logbook-logstash-logback-encoder/src/main/java/org/zalando/logbook/lle/LogbackLogstashSink.java

```diff
@@ -0,0 +1,49 @@
+package org.zalando.logbook.lle;
+
+import java.io.IOException;
+
+import org.slf4j.Marker;
+import org.zalando.logbook.Correlation;
+import org.zalando.logbook.HttpLogFormatter;
+import org.zalando.logbook.HttpRequest;
+import org.zalando.logbook.HttpResponse;
+import org.zalando.logbook.Precorrelation;
+import org.zalando.logbook.Sink;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public final class LogbackLogstashSink implements Sink {
+
+    private final HttpLogFormatter formatter;
+    private final LogbackLogstashLogWriter logWriter;
+
+    @Override
+    public boolean isActive() {
+        return logWriter.isActive();
+    }
+
+    @Override
+    public void write(final Precorrelation precorrelation, final HttpRequest request) throws IOException {
+        Marker marker = new AutodetectPrettyPrintingMarker(request.getScheme(), formatter.format(precorrelation, request));
```

Comment:
I'd just hardcode `http` as the field name. Wouldn't it be confusing anyway that sometimes this field is called `http` and sometimes `https`?

Reply:
Agreed